build/build.cake

```diff
@@ -257,19 +257,20 @@ Task("Package")
     .IsDependentOn("InheritDoc")
     .Does(() =>
 {
-    // Invoke the pack target to generate the code to be packed.
-    MSBuildSolution("Pack", ("GenerateLibraryLayout", "true"), ("PackageOutputPath", nupkgDir));
-
-    foreach (var nuspec in GetFiles("./*.nuspec"))
-    {
-        var nuGetPackSettings = new NuGetPackSettings
-        {
-            OutputDirectory = nupkgDir,
-            Version = Version
-        };
-
-        NuGetPack(nuspec, nuGetPackSettings);
-    }
+    // // Invoke the pack target to generate the code to be packed.
+    // MSBuildSolution("Pack", ("GenerateLibraryLayout", "true"), ("PackageOutputPath", nupkgDir));
+
+    // foreach (var nuspec in GetFiles("./*.nuspec"))
+    // {
+    //     var nuGetPackSettings = new NuGetPackSettings
+    //     {
+    //         OutputDirectory = nupkgDir,
+    //         Version = Version
+    //     };
+
+    //     NuGetPack(nuspec, nuGetPackSettings);
+    // }
+    StartPowershellFile("./Nuget-prep.ps1");
```

Comment:
>StartPowershell [](start = 4, length = 15)

Commented code should be removed. And you should add a comment explaining the intention. It looks like this Task no longer does what its description claims it does.